client/src/components/Alert.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from "react";
+import { Snackbar } from "@material-ui/core";
+import MuiAlert from "@material-ui/lab/Alert";
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+function AlertMessage(props) {
+  const [open, setOpen] = useState(false);
```

Comment:
Should open be a prop rather than state here so that it can be controlled from a higher order component?

Reply:
It looks like you could use props.alert.error in place of having an additional state object

Reply:
I also need open as a state in a handleClose function, whenever "x" is clicked on a pop-up window. Because a fact of closing a pop-up message doesn't set error to false

Reply:
Yeah, that's a fair point. Okay resolving this one